### PR TITLE
feat: custom the grpc gateway restful path

### PR DESCRIPTION
**Pull request by LinuxSuRen**
> We highly recommend you read [the contributor's documentation](https://github.com/LinuxSuRen/api-testing/blob/master/CONTRIBUTING.md) before starting the review process especially since this is your first contribution to this project.
>
> It was updated on 2024/5/27

**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

File: console/atest-ui/src/views/net.ts [hunk 1]
```diff
@@ -352,14 +336,13 @@ function GenerateCode(request: GenerateRequest,
       Generator: request.generator
     })
   }
-  fetch('/server.Runner/GenerateCode', requestOptions)
+  fetch(`/api/v1/code-generators/generate`, requestOptions)
```
Comment (yuluo-yx) [inline console/atest-ui/src/views/net.ts:339]: IMO, here we can all use camel-named hair to keep the format of the uri consistent

`\xxx\codeGenerators\xxx`

File: console/atest-ui/src/views/net.ts [hunk 2]
```diff
@@ -352,14 +336,13 @@ function GenerateCode(request: GenerateRequest,
...
     .then(DefaultResponseProcess)
     .then(callback).catch(errHandle)
 }
 
 function ListCodeGenerator(callback: (d: any) => void, errHandle?: (e: any) => void | null) {
-  fetch('/server.Runner/ListCodeGenerator', {
-    method: 'POST',
+  fetch('/api/v1/code-generators', {
```
Comment (yuluo-yx) [inline console/atest-ui/src/views/net.ts:345]: ditto

File: console/atest-ui/src/views/net.ts [hunk 3]
```diff
@@ -391,9 +373,10 @@ function CreateOrUpdateStore(payload: any, create: boolean,
     body: JSON.stringify(payload)
   }
   
-  let api = '/server.Runner/CreateStore'
+  let api = '/api/v1/stores'
```
Comment (yuluo-yx) [inline console/atest-ui/src/views/net.ts:376]: Does this variable appear to be unused?

File: console/atest-ui/src/views/net.ts [hunk 4]
```diff
@@ -467,15 +445,11 @@ function GetSecrets(callback: (d: any) => void, errHandle?: (e: any) => void | n
...
     headers: {
       'X-Auth': getToken()
-    },
-    body: JSON.stringify({
-      name: filter
-    })
+    }
   }
-  fetch('/server.Runner/FunctionsQuery', requestOptions)
+  fetch(`/api/v1/functions?name=${filter}`, requestOptions)
```
Comment (yuluo-yx) [inline console/atest-ui/src/views/net.ts:452]: Can we standardize here? It's all about restful style and avoiding the plain way of passing parameters.

File: console/atest-ui/src/views/net.ts [hunk 5]
```diff
@@ -524,16 +498,12 @@ function CreateOrUpdateSecret(payload: any, create: boolean,
...
       'X-Store-Name': Cache.GetCurrentStore().name,
       'X-Auth': getToken()
-    },
-    body: JSON.stringify({
-      name: name
-    })
+    }
   }
-  fetch('/server.Runner/GetSuggestedAPIs', requestOptions)
+  fetch(`/api/v1/suggested-apis?name=${name}`, requestOptions)
```
Comment (yuluo-yx) [inline console/atest-ui/src/views/net.ts:506]: ditto

File: pkg/util/net.go [hunk 6]
```diff
@@ -21,9 +22,12 @@ import (
 
 // GetPort returns the port of the listener
 func GetPort(listener net.Listener) string {
+<<<<<<< HEAD
```
Comment (yuluo-yx) [inline pkg/util/net.go:25]: Looks like there's some problems.

File: pkg/util/net.go [hunk 7]
```diff
@@ -21,9 +22,12 @@ import (
...
+=======
 	if listener == nil {
 		return ""
 	}
+>>>>>>> origin/master
```
Comment (yuluo-yx) [inline pkg/util/net.go:30]: ditto

File: console/atest-ui/src/views/net.ts (hunk 3 above)
Comment (LinuxSuRen) [reply console/atest-ui/src/views/net.ts:376]: It's useful. See line 377

File: console/atest-ui/src/views/net.ts (hunk 4 above)
Comment (LinuxSuRen) [reply console/atest-ui/src/views/net.ts:452]: Of course, I will change the API style in this PR.

File: pkg/util/net.go (hunk 6 above)
Comment (LinuxSuRen) [reply pkg/util/net.go:25]: Yes, I missed this one.

**Review by LinuxSuRen — COMMENTED** [review]
Thanks for the review. I didn't finish it yet.

File: console/atest-ui/src/views/net.ts (hunk 4 above)
Comment (yuluo-yx) [reply console/atest-ui/src/views/net.ts:452]: got it

**Review by yuluo-yx — APPROVED** [review]
lgtm.

**Pull request merged**